actionview/test/template/sanitize_helper_test.rb

```diff
@@ -40,4 +40,12 @@ def test_strip_tags_will_not_encode_special_characters
   def test_sanitize_is_marked_safe
     assert_predicate sanitize("<html><script></script></html>"), :html_safe?
   end
+
+  def test_sanitized_allowed_tags_class_method
+    assert_nothing_raised { self.class.sanitized_allowed_tags }
```

Comment:
We should assert the return of `self.class.sanitized_allowed_tags` instead of using `assert_nothing_raised`.

Reply:
Hello @rafaelfranca . I just noticed that you removed this PR from 6.1.0 milestone. Can you give some comments on why this PR can't progress?